src/scribe_data/cli/list.py

```diff
@@ -0,0 +1,153 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Union
+
+# Load language metadata from JSON file
+METADATA_FILE = Path(__file__).parent.parent / 'resources' / 'language_meta_data.json'
+
+def load_language_metadata() -> Dict:
+    with METADATA_FILE.open('r', encoding='utf-8') as file:
+        return json.load(file)
+
+LANGUAGE_METADATA = load_language_metadata()
+LANGUAGE_MAP = {lang['language'].lower(): lang for lang in LANGUAGE_METADATA['languages']}
+
+DATA_DIR = Path('scribe_data_json_export')
+
+def print_formatted_data(data: Union[Dict, List], word_type: str) -> None:
+    if not data:
```

Comment:
Let's put a vertical space between the if-else statements so it's a bit easier to read them 🤓

Reply:
So:

```py
if something:
   True
# <- Space here so it's easy to see where each condition ends. 
else:
    False
```